devops/yarn/checkstyle.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0"?>
```

Comment:
How about just keep one copy of checkstyle.xml whole ray project, and you could config the checkstyle.xml path in the pom.xml, so we could just maintain just one copy here.

Reply:
Ok, I just want to maintain the 'ray on yarn' to be a independent folder. Anyway, maybe I can use the checkstyle.xml in 'java' folder.